### ISSUE TITLE
[Flipper] Add `Save Results` button 

### ISSUE THREAD

**Issue by troZee** [bamlab/android-performance-profiler#50]
### Feature Description 

Saving a results of experiment is not supported inside a Flipper.  
`Save Result` button could be next to `Reset` button in one row 
<img width="374" alt="Screenshot 2022-10-31 at 10 37 04" src="https://user-images.githubusercontent.com/12766071/198978156-7ad4e576-97b8-4b7d-98d1-68250c157667.png">

Once user clicked on that, it could save all result into JSON file. Under the hood, we could use below approach 
```
  const { writeResults } = await measurePerformance(bundleId, testCase);
  writeResults();
```

Right now, saving a results is only supported by e2e tests. 

### Expected outcome 
- Click on `Save Results` button 
- Choose json dir to save 
- Click on save button in the prompt 
- Create new JSON file with all results in previously selected dir 

### Supported Platform
- Flipper

**Comment by Almouro** [comment]
Thanks for suggesting this! I agree this is a must have!
We should be able to pick it up soon 🤞

**Comment by Almouro** [comment]
Finally done in #61 and released in 0.4.0 of the Flipper plugin!
